test/certs/ee-self-signed.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
We've tended to not only commit the requisite certificates, but also the "mkcert" invocation that creates them into `test/certs/setup.sh`.  Not sure how closely that's been followed, but if not too much trouble...

Reply:
The cert had been provided by a bug reporter in https://github.com/openssl/openssl/issues/1418#issuecomment-562891841.
In this case it was no big deal to add a call to `setup.sh` for producing a suitable fresh test cert; done.

Reply:
Thanks!